### DISCUSSION TITLE
Welcome to UnblockNeteaseMusic/server-rust Discussions!

### DISCUSSION THREAD

**Discussion by 1715173329** [UnblockNeteaseMusic/server-rust#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by banmujun** [comment]
我使用了docker的镜像，发现会报错：WARN:(spawn)[yt-dlp][stderr]WARNING:[youtube:search]<urlopen error timed out>.
是因为docker镜像还不能使用YtDl吗？
